crates/bevy_app/src/app_builder.rs

```diff
@@ -241,6 +319,22 @@ impl AppBuilder {
         self
     }
 
+    /// Inserts a non-send resource to the app
+    ///
+    /// You usually want to use `insert_resource`, but there are some special cases when a resource must
```

Comment:
This should reference the `Send` trait, and perhaps give a more concrete example or two.